staging/cse/windows/azurecnifunc.ps1

```diff
@@ -416,6 +417,22 @@ function New-ExternalHnsNetwork
         {
             $managementIP = $netIP.IPAddress

```

Comment:
this was just logged and not used before right? Do we need a more strict filter than just first adapter with an ip?

Reply:
It is used to create the network `ext` in https://github.com/Azure/AgentBaker/blob/ed90c97f0ab41ce8c237ca113114bcfe44ca3283/staging/cse/windows/azurecnifunc.ps1#L471C6-L471C30.
I think that it was provided by Windows team and was added in supporting accelerated network three years ago